rewrite-core/src/test/java/org/openrewrite/text/AppendToTextFileTest.java

```diff
@@ -157,6 +158,48 @@ void multipleFiles() {
         );
     }
 
+    // This test is just to demonstrate RewriteRun defect, it's expected to fail since new file `file1.txt` generated but pass.
+    // todo, kunli, to be removed after this PR.
+    @Disabled("Expected to be failed since new file generated")
+    @Test
+    void demonstrateRewriteRunDefect() {
+        rewriteRun(
```

Comment:
JUnit has an assertion method which accepts a lambda and asserts that the lambda will throw an exception. If this is expected to fail then we can wrap it in `Assertions.assertThrows()` and remove the `@Disabled`

Reply:
good point, I thought I should remove this test after this PR,  but good to keep it, updated to use `assertThrows`.